controllers/smsController.js

```diff
@@ -0,0 +1,21 @@
+const Message = require('../models/smsModel')
+
+const saveMessage = async (req, res) => {
```

Comment:
Here https://github.com/Open-Avenues-Foundation/P21020-2/pull/9/files#diff-c3c477ada96cd37ae262078f88e6ed77dba5d8e724921d853348ef31507b985bR4 `saveMessage` only receives one parameter. The parameters in this function should match, what about `const saveMessage = async (message) => { ...`